### PR TITLE
Update keyboard layout files, images, and landing page

### PR DESCRIPTION
**Pull request by GwynethM**
Update to include Windows 8, 8.1 and 10 data.  Existing only had up to Windows 7.

### CODE REVIEW

**Comment by Khairunj** [comment]
OPS Build status updates of commit _[9df62c4](https://github.com/GwynethM/globalization/pull/74/commits/9df62c4f1c2fc1ca603f484e2256fc900e7cffb0)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by Khairunj** [comment]
OPS Build status updates of commit _[9df62c4](https://github.com/GwynethM/globalization/pull/74/commits/9df62c4f1c2fc1ca603f484e2256fc900e7cffb0)_: 

### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/MicrosoftDocs/globalization, working directory: W:\00o0-s, target branch: master, target branch commit: cb3e0c3f44addd5b56b214182469c1078370b07b, source branch: 4e44cc3041b6a4ef2e3ca55dec5a6979/master, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C2%5C7e74dcc8-0c29-2d83-8948-de96dd917559%5CPullRequest%5C201901021558116894-74%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fKAoaHrceTRnyg%2FnIutqQtULMoFje3bfFBqgJY%2FthgI%3D&st=2019-01-02T15%3A53%3A27Z&se=2019-02-02T15%3A58%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Khairunj** [comment]
OPS Build status updates of commit _[4f7a4d3](https://github.com/GwynethM/globalization/pull/74/commits/4f7a4d3e4fbeaf5fc09898b58605015cecccc36c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by Khairunj** [comment]
OPS Build status updates of commit _[4f7a4d3](https://github.com/GwynethM/globalization/pull/74/commits/4f7a4d3e4fbeaf5fc09898b58605015cecccc36c)_: 

:clock10: **Incremental building**: average incremental building time is _47 sec(s)_

**Comment by Khairunj** [comment]
OPS Build status updates of commit _[4f7a4d3](https://github.com/GwynethM/globalization/pull/74/commits/4f7a4d3e4fbeaf5fc09898b58605015cecccc36c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[globalization/input/font-support.md](https://github.com/GwynethM/globalization/blob/master/globalization/input/font-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/input/font-support?branch=pr-en-us-74) |
[globalization/input/font-technology.md](https://github.com/GwynethM/globalization/blob/master/globalization/input/font-technology.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/input/font-technology?branch=pr-en-us-74) |
[globalization/keyboards/images/0000K.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/0000K.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/0000k.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/0000KW.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/0000KW.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/0000kw.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A0.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A0.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a0.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A1.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A1.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a1.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A2.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A2.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a2.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A3.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A3.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a3.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A3W.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A3W.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a3w.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A4.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A4.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a4.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A4W.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A4W.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a4w.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A5.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A5.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a5.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A6.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A6.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a6.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A6W.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A6W.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a6w.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A7.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A7.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a7.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A7W.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A7W.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a7w.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A8.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A8.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a8.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A8K.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A8K.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a8k.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A8KW.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A8KW.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a8kw.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00A9.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00A9.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00a9.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00AA.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00AA.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00aa.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00ABK.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00ABK.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00abk.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00AC.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00AC.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00ac.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00ACK.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00ACK.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00ack.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00ACKW.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00ACKW.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00ackw.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00ACW.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00ACW.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00acw.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00AD.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00AD.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00ad.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00AE.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00AE.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00ae.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00AF.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00AF.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00af.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B0.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B0.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b0.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B0K.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B0K.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b0k.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B0KW.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B0KW.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b0kw.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B1.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B1.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b1.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B2.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B2.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b2.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B2W.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B2W.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b2w.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B3.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B3.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b3.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B3W.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B3W.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b3w.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B4.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B4.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b4.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B4K.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B4K.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b4k.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B4KW.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B4KW.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b4kw.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B5.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B5.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b5.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B5W.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B5W.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b5w.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B6.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B6.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b6.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B7.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B7.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b7.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B7K.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B7K.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b7k.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B8.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B8.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b8.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B8K.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B8K.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b8k.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00B9.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00B9.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00b9.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00BA.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00BA.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00ba.jpg?branch=pr-en-us-74) |
[globalization/keyboards/images/00BAW.jpg](https://github.com/GwynethM/globalization/blob/master/globalization/keyboards/images/00BAW.jpg) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/globalization/keyboards/images/00baw.jpg?branch=pr-en-us-74) |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C2%5C7e74dcc8-0c29-2d83-8948-de96dd917559%5CPullRequest%5C201901021602513649-74%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Qwc5VNuTtkoAiQa8f9S1%2FTDU8LweT6sl3D9r9ACQw4s%3D&st=2019-01-02T16%3A00%3A04Z&se=2019-02-02T16%3A05%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: globalization/input/font-technology.md [hunk 1]
```diff
@@ -79,14 +79,14 @@ The Windows operating system also allows enabling font substitution but this sho
...
 1.  **Do not assume a selected font supports the desired script.** For example, it is impossible to use Miriam (a Hebrew font) to represent the hiragana script.
 2.  **Do not hardcode font face names.** These values may need to change based on the language of the UI and should be defined as a table or matrix based on the required script.
-3.  **Understand your font fallback.** Define to meet your design goals. For example, using HTML formats, you could define font-family: Georgia, "Times New Roman", "Microsoft YaHei", 微软雅黑, STXihei, 华文细黑, serif; which lists the Latin fonts first, includes both Microsoft and Apple fonts, and includes Chinese font names. For other coding solutions, the similar logic will be in a table or matrix.
+3.  **Understand your font fallback.** Define to meet your design goals. For example, using HTML formats, you could define font-family: Georgia, "Times New Roman", "Microsoft YaHei", <span style="lang:zh-HanS">微软雅黑</span>, STXihei, <span style="lang:zh-HanS">华文细黑</a>, serif; which lists the Latin fonts first, includes both Microsoft and Apple fonts, and includes Chinese font names. For other coding solutions, the similar logic will be in a table or matrix.
```
Comment (nschonni) [inline globalization/input/font-technology.md]: `lang` is a sudo selector for CSS not a property. Think this should be `<span lang=zh-Hans">`. Addit it is a good idea :+1:

Comment (GwynethM) [reply globalization/input/font-technology.md]: I tried that and it didn't work (at least not in the GitHub preview environment). It just showed the tag on the page without doing the formatting. I'm open to suggestions! :)

Comment (nschonni) [reply globalization/input/font-technology.md]: Setting the `lang` is more for screen readers and machines so they understand what language a block is in that differs from the top level `<html lang="en">` usually at the top of the page.
I don't think GitHub will render a span differently

**Pull request merged**